samples/snippets/src/test/java/aiplatform/DeleteFeaturestoreSampleTest.java

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
nit. This test may not be needed, assuming detele is well covered with all test tear down.